charmcraft.yaml

```diff
@@ -94,10 +97,6 @@ config:
       description: Domains that need to be excluded from proxying no_proxy="test.com,test.co.uk", it is a comma separate list
       type: string
       default: ''
-    smtp_connection_uri:
-      description: SMTP connection URI
-      type: string
-      default: 'smtps://test:test@mailslurper:1025/?skip_ssl_verify=true'
```

Comment:
I removed this config option as it had no validation, in favor of using the new integration. I think any existing Kratos deployment only uses the default value because the smtp server is not used unless you enable local identity provider, which will be released in a new version.